File: src/test/samples/invalidSamples/invalidSample.pas
//#region FirstRegion
program HelloWorld;
begin
  writeln('Hello, World!');
end.
// #endregion

(*#endregion Invalid end boundary*)
{#region Invalid start boundary}

{  #region    Second Region  }
procedure MyProcedure;
begin
  (*    #region    InnerRegion    *)
  writeln('Inside region');
  (*  #endregion   ends InnerRegion  *)

  {#region}
  writeln('Another unnamed region');
  {#endregion}
end;
{   #endregion   }

File: src/test/samples/invalidSamples/invalidSample.ml
(*#region FirstRegion*)
let x = 42
(*#endregion*)

(* #endregion Invalid end boundary *)
(* #region Invalid start boundary *)

(*  #region Second Region *)
let my_function () =
  (*  #region    InnerRegion   *)
  Printf.printf "Hello, World!\n"
  (*  #endregion   ends InnerRegion *)

  (* #region *)
  let y = 24 in ()
  (* #endregion *)
(*   #endregion *)
